services/eventsapi/main.py

```diff
@@ -1,4 +1,5 @@
-from typing import Optional
+from tokenize import String
```

Comment:
This file has a few minor lint errors from `flake8` (there's no CI set up in this repo to catch these, but it'd be nice to keep things linted locally)